src/main/java/com/codessquad/qna/controller/AnswerController.java

```diff
@@ -0,0 +1,46 @@
+package com.codessquad.qna.controller;
+
+import com.codessquad.qna.domain.Answer;
+import com.codessquad.qna.domain.Question;
+import com.codessquad.qna.domain.User;
+import com.codessquad.qna.exception.FailedUserLoginException;
+import com.codessquad.qna.service.AnswerService;
+import com.codessquad.qna.service.QuestionService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import javax.servlet.http.HttpSession;
+
+import static com.codessquad.qna.controller.HttpSessionUtils.getSessionUser;
+import static com.codessquad.qna.controller.HttpSessionUtils.isLoginUser;
+
+@Controller
+@RequestMapping("/questions/{questionId}/answers")
+public class AnswerController {
+    private final Logger logger = LoggerFactory.getLogger(UserController.class);
+    private final QuestionService questionService;
+    private final AnswerService answerService;
+
+    @Autowired
+    public AnswerController(QuestionService questionService, AnswerService answerService) {
+        this.questionService = questionService;
+        this.answerService = answerService;
+    }
+
+    @PostMapping("")
```

Comment:
빈 문자열을 두시지 말고 그냥 과감히 빼셔도 괜찮습니다!

```suggestion
    @PostMapping